packages/visualizations/addon/chart-builders/dimension.js

```diff
@@ -109,11 +109,11 @@ export default Ember.Object.extend({
         //Handling the case when some of the data group does not exist
         if (byDim.getDataForKey(s) && byDim.getDataForKey(s).length) {
           // Extending the data for date with the grouped dimension and metric value
-          $.extend(dataForDate, { [seriesName[index]]: byDim.getDataForKey(s)[0][metric] });
+          Object.assign(dataForDate, { [seriesName[index]]: byDim.getDataForKey(s)[0][canonicalizeMetric(metric)] });
```

Comment:
So unlike `$.extend`. `Object.assign` returns a new object. So you would have to return these.. what's going on that this works?

Reply:
I set some breakpoints before and after this `Object.assign` line and it seems that it is changing the `dataForDate` variable. I think it would return a new object if the first argument was anonymous.